CODEOWNERS

```diff
@@ -100,14 +100,14 @@ CODEOWNERS @prestodb/team-tsc
 # Presto core
 
 # Presto analyzer and optimizer
-/presto-main/src/*/java/com/facebook/presto/sql @jaystarshot @feilong-liu @ClarenceThreepwood @prestodb/committers
```

Comment:
Not sure what's happening here but the CODEOWNERS shouldn't be changing in this PR. This did recently change so maybe you just need to rebase?